src/NJsonSchema.CodeGeneration.TypeScript/Templates/ConvertToClassTemplate.tt

```diff
@@ -9,7 +9,7 @@ if (<#=Model.Value#> && <#=Model.Value#>.constructor === Array) {
 <#if(Model.IsArrayItemNewableObject){#>
         <#=Model.Variable#>.push(<#=Model.ArrayItemType#>.fromJS(item<#if(Model.HandleReferences){#>, _mappings<#}#>));
 <#}else{#>
-<#if(Model.IsArrayItemDate){#>
+<#if(Model.IsArrayItemDate || Model.IsArrayItemDateTime){#>
         <#=Model.Variable#>.push(<#=Model.StringToDateCode#>(item));
```

Comment:
Maybe we should also add `StringToDateTimeCode` (like in the other direction) - it may be more (redundant) code, but it may be clearer for the next who looks into this code?

Reply:
Or I could put a comment on `StringToDateCode` that it is used for both Date and DateTime conversion?
I don't think there would be a reason that these would need to be different.
I'm happy either way, let me know.